README.md

```
@@ -3,94 +3,111 @@
+bun run deploy:pages # Build and deploy to Cloudflare Pages via Wrangler
 ```
 
-> **Note**
->
-> Next.js 16 enables Google Fonts by default. The repository configures Turbopack to reuse system TLS certificates during builds,
-> which keeps static exports working reliably in headless CI environments.
+---
 
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)